lib/component.js

```diff
@@ -327,6 +327,12 @@ class Component extends WebComponent {
     }
     this.initializing = true;
 
+    for (const attrsSchemaKey in this._attrsSchema) {
+      if (!Object.prototype.hasOwnProperty(this._attrs, attrsSchemaKey) && this._attrsSchema[attrsSchemaKey].required) {
+        throw new Error(`Missing required attribute '${attrsSchemaKey}' for ${this}`);
```

Comment:
```suggestion
        throw new Error(`${this}: Missing required attr '${attrsSchemaKey}'`);
```

mostly so format is consistent with other errors